src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -843,9 +843,9 @@ public override Task CloseAsync()
 #else
         public Task CloseAsync()
 #endif
-            => Close(connectionClosing: false, async: true);
+            => Close(connectionClosing: false, async: true, cancellationToken: default);
 
-        internal async Task Close(bool connectionClosing, bool async)
+        internal async Task Close(bool connectionClosing, bool async, CancellationToken cancellationToken = default)
```

Comment:
Close is special - since it's very similar to Dispose, the System.Data API does not accept a cancellation token. Not sure that this method needs to accept it (is there any need?)

Reply:
On second thoughts - yeah, there is no need (at least for now)